src/screens/DetailDomain/elements/DetailDomainScreenHeader.js

```diff
@@ -137,7 +138,7 @@ const styles = StyleSheet.create({
     paddingRight: 20
   },
   followerNumber: {
-    color: COLORS.blackgrey,
+    color: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change seems to be part of an effort to update the color scheme of the application. As long as this change is consistent with the new color scheme, it should be fine.


<!-- This is an auto-generated comment by OSS CodeRabbit -->